src/CoreManager.js

```diff
@@ -393,10 +393,9 @@ module.exports = {
 
   setLiveQueryController(controller: any) {
     requireMethods('LiveQueryController', [
```

Comment:
can you help me understand what this change is please?

Reply:
Those function (subscribe, unsubscribe, open, close) already exists in the LiveQueryClient.

In the LiveQueryController those were just wrapper functions. Instead of using wrapper functions call the ones from the LiveQueryClient.

setDefaultLiveQueryClient, getDefaultLiveQueryClient, _clearCachedDefaultClient already existed but weren't documented.